app/static/js/submitform.js

```diff
@@ -0,0 +1,14 @@
+function make_request() { 
+    let queryField = document.getElementById('queryField').value;
+    let limitField = document.getElementById('limitField').value;
+    const http = new XMLHttpRequest();
+    const queryUrl = encodeURIComponent(`${queryField}`)
+    const limitUrl = encodeURIComponent(`${limitField}`)
+    url = `${window.location.origin}/api/player?query=${queryUrl}&limit=${limitUrl}`
+    http.open('GET', url);
```

Comment:
What if we used fetch instead? https://developers.google.com/web/updates/2015/03/introduction-to-fetch
```
fetch(url).then(function(response) {
  parsedResponseJSON = JSON.stringify(response.json()), undefined, 2);
  document.getElementById('playerResponse').innerHTML = parsedResponseJSON;
}
```

Reply:
done